Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -221,12 +230,14 @@ public void centerJunctionTop(GrabberCameraPipeline pipeline) {
         double xOffset, yOffset;
 
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
-            yOffset = Constants.CAMERA_CENTER_Y - pipeline.yPosition;
+            xOffset = pipeline.xPosition-Constants.CAMERA_CENTER_X;
+            yOffset = Constants.CAMERA_CENTER_Y-pipeline.yPosition;
 
             // center the cone on the junction top
             if (pipeline.detected) {
                 driveWithIMU(junctionTopPixelsMotorPower(xOffset), junctionTopPixelsMotorPower(yOffset), 0.0);
+                telemetry.addData("x", pipeline.xPosition-Constants.CAMERA_CENTER_X);
+                telemetry.addData("y", Constants.CAMERA_CENTER_Y-pipeline.yPosition);
```

Comment:
Use the variables you made.
```suggestion
                telemetry.addData("x", xOffset);
                telemetry.addData("y", yOffset);
```

Reply:
Changed

Reply:
This one too

Reply:
Reset